packages/commercetools/composables/nuxt/index.js

```diff
@@ -0,0 +1,11 @@
+import path from 'path';
+
+export default function (moduleOptions) {
+  if (!moduleOptions.disableMiddleware && this.options.router && this.options.router.middleware && !this.options.router.middleware.includes('commercetools')) {
```

Comment:
don't do such a long ifs, split them into smaller functions that tell and mean something, for example in this case i can see one that can be split:

```
const hasDefinedMiddleware = (options) => options.router && options.router.middleware && options.router.middleware.includes('commercetools')
```

then 
```
if (!moduleOptions.disableMiddleware && !hasDefinedMiddleware(this.options)) 
```
cleaner?